### ISSUE TITLE
`BrowserCompat` widget should break out mobile Chrome separately

### ISSUE THREAD

**Issue by tomayac** [GoogleChrome/web.dev#8807]
For [`ContactManager`](https://developer.mozilla.org/en-US/docs/Web/API/ContactsManager/select#browser_compatibility), compare… 
![Screenshot 2022-10-06 at 11 49 38](https://user-images.githubusercontent.com/145676/194283359-9040dfae-53ef-439d-a2c1-3196e4a6adef.png)

…to…

![Screenshot 2022-10-06 at 11 53 13](https://user-images.githubusercontent.com/145676/194283584-cb0f387d-b180-44c2-9583-8532c59b302c.png)

**Comment by rachelandrew** [comment]
I don't think we want this for everything, given that for many features Chrome support is across Chrome/Android, and WebView. It would be useful to have as an option (or automatically only if there are differences in support between the three).

**Comment by tomayac** [comment]
Having a flag would work indeed, and reduce clutter. Maybe something like:


```nunjucks
<!-- Default -->
{% BrowserCompat 'example.API' %}
<!-- Customized -->
{% { BrowserCompat 'example.API', platforms: 'desktop, mobile' } %}
```

**Comment by jpmedley** [comment]
To provide some context. The reason for this request is that many capabilities arrive on different platforms in different versions because reaching beyond the browser to platform capabilities requires writing code that is unique to each platform.

**Comment by tomayac** [comment]
Still worthwhile to have.
